VersionHistory.md

```diff
@@ -7,6 +7,8 @@ is updated in preparation for publishing an updated NuGet package.
 
 Prefix the description of the change with `[major]`, `[minor]` or `[patch]` in accordance with [SemVer](http://semver.org).
 
+* [minor] Add `NullTraceProvider`.
```

Comment:
This release note would indicate a public `NullTraceProvider`, not a subtle change to `GetProvider`.